### ISSUE TITLE
Is that possible to enlarge the segment ID during fine-tune?

### ISSUE THREAD

**Issue by xyzhou-puck** [google-research/bert#127]
Hi,

I have two questions:

1) Is that possible to enlarge the segment ID during fine-tune? 

2) Is that possible to concat the positional-encoding and segment-embedding as one vector representation instead of summing up during input?

Thanks,
Xiangyang

**Comment by jacobdevlin-google** [comment]
You can't change the model in any way without creating a mismatch with pre-training. If you're OK with creating a mismatch with pre-training, you can change the model however you like.

For the second one I would not recommend doing that unless you're planning on pre-training from scratch (but even then I don't really understand the advantage).
